js/dom.js

```diff
@@ -0,0 +1,92 @@
+// eslint-disable-next-line import/extensions
+import { Book, myLibrary } from './library.js';
+
+const form = document.querySelector('#form');
+
+const clearElement = (el) => {
+  while (el.firstChild) el.removeChild(el.firstChild);
+};
+
+const renderForm = () => {
+  form.classList.toggle('hide');
+};
+
+const render = () => {
+  const div = document.querySelector('.work');
+
+  clearElement(div);
+  // eslint-disable-next-line no-use-before-define
+  renderBooks(div);
+};
+
+function addBookToLibrary() {
+  const title = document.querySelector('#title').value;
+
+  const author = document.querySelector('#author').value;
+
+  const pages = document.querySelector('#pages').value;
+
+  const read = document.querySelector('#read').checked;
+  if (title.length > 0 && author.length > 0 && pages.length > 0) {
+    const newbook = new Book(title, author, pages, read);
+
+    myLibrary.push(newbook);
+
+    renderForm();
+    render();
+  } else {
+    // eslint-disable-next-line no-alert
+    window.alert('Title/Author/Pages must not be empty');
```

Comment:
Opps 🤦‍♂️ , This should be production code, we shouldn't have `alert`, please remove the `alert`.
If you don't want to pop-up new div message, I think you can make use of `required` on the inputs tags.